src/main/scala/fs2/aws/kinesis/kpl.scala

```diff
@@ -0,0 +1,44 @@
+package fs2
+package aws
+package kinesis
+
+import com.google.common.util.concurrent.{ListenableFuture, FutureCallback, Futures}
+import cats.effect.{Effect, Concurrent}
+import fs2.aws.internal.Internal._
+import com.amazonaws.services.kinesis.producer.UserRecordResult
+import scala.concurrent.ExecutionContext
+
+
+package object publisher {
+  def writeToKinesis[F[_]](
```

Comment:
maybe we should start adding Javadocs, I can do it on the S3 functions if you want to do it for the kinesis ones, and going forward we should document all the interfaces

Reply:
sounds good!